### PR TITLE
Add build badge + rework readme

### PR DESCRIPTION
**Pull request by cewert**
I just wanted to add a build badge 🤷 

## Changes
- Add build badge that shows latest unstable build status and links the unstable workflows (for easy access to build zips for install)
- Converted the badge HTML to markdown ( left the branding stuff at top because there's no way to center things without HTML)
  - The badges are no longer centered but now left aligned and wrap
- Reworked the main text to hopefully be a bit easier to read and also help people get a beta/unstable release running

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
+[![Build Status](https://img.shields.io/github/actions/workflow/status/jellyfin/jellyfin-roku/build-dev.yml?logo=github&branch=unstable "Build Status")](https://github.com/jellyfin/jellyfin-roku/actions/workflows/build-dev.yml?query=branch%3Aunstable)
+[![Current Release](https://img.shields.io/github/release/jellyfin/jellyfin-roku.svg?logo=github "Current Release")](https://github.com/jellyfin/jellyfin-roku/releases)
+[![Translation Status](https://translate.jellyfin.org/widgets/jellyfin/-/jellyfin-roku/svg-badge.svg "Translation Status")](https://translate.jellyfin.org/projects/jellyfin/jellyfin-roku/?utm_source=widget)
+[![Matrix](https://img.shields.io/matrix/jellyfin:matrix.org.svg?logo=matrix "Chat on Matrix")](https://matrix.to/#/#jellyfin-dev-roku:matrix.org)
+[![Reddit](https://img.shields.io/badge/reddit-r%2Fjellyfin-%23FF5700.svg?logo=reddit "Join our Subreddit")](https://www.reddit.com/r/jellyfin)
+[![License](https://img.shields.io/github/license/jellyfin/jellyfin-roku.svg "GPL 2.0 License")](LICENSE)
 
-## Getting Started
+The Jellyfin Roku App is a Jellyfin client for Roku Devices and is still very much a work in progress. Please get involved if you can!
```
Comment (1hitsong) [inline README.md]: >  and is still very much a work in progress.

Do y'all think we need to still include this text? I can understand having this when we were first getting up and running, but we're now 3+ years into the client.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
+
+> NOTE: The beta app will always be at the bottom of your Roku's channel list and it will *not* automatically update.
+
+## Advanced
+
+For more advanced deployment methods, access to crash logs, or to learn how to setup a developer environment so you can write some code yourself please read the [DEVGUIDE](DEVGUIDE.md).
+
+## Feature Requests
+
+New feature requests are always welcome but before creating an issue please read though the [existing issues](https://github.com/jellyfin/jellyfin-roku/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) to see if someone has already raised one for what you're looking for.
```
Comment (cewert) [inline README.md]:
```suggestion
New feature requests are always welcome but before creating an issue please read through the [existing issues](https://github.com/jellyfin/jellyfin-roku/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) to see if someone has already raised one for what you're looking for.
```

File: README.md (hunk 1 above)
Comment (cewert) [reply README.md]: ~~Changed this line to `Jellyfin Roku is the official Jellyfin client for Roku. Please get involved if you can!` how's that sound?~~

Changed this line to `Jellyfin Roku is the official Jellyfin client for Roku devices. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start.` How's that sound? Looked at other jellyfin repos for inspiration and this is how the [android-tv repo](https://github.com/jellyfin/jellyfin-androidtv) does it

Comment (1hitsong) [reply README.md]: I like it!

File: README.md [hunk 3]
```diff
@@ -1,39 +1,41 @@
...
+[![Current Release](https://img.shields.io/github/release/jellyfin/jellyfin-roku.svg?logo=github "Current Release")](https://github.com/jellyfin/jellyfin-roku/releases)
+[![Translation Status](https://translate.jellyfin.org/widgets/jellyfin/-/jellyfin-roku/svg-badge.svg "Translation Status")](https://translate.jellyfin.org/projects/jellyfin/jellyfin-roku/?utm_source=widget)
+[![Matrix](https://img.shields.io/matrix/jellyfin:matrix.org.svg?logo=matrix "Chat on Matrix")](https://matrix.to/#/#jellyfin-dev-roku:matrix.org)
+[![Reddit](https://img.shields.io/badge/reddit-r%2Fjellyfin-%23FF5700.svg?logo=reddit "Join our Subreddit")](https://www.reddit.com/r/jellyfin)
+[![License](https://img.shields.io/github/license/jellyfin/jellyfin-roku.svg "GPL 2.0 License")](LICENSE)
 
-## Getting Started
+Jellyfin Roku is the official Jellyfin client for Roku devices. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start.
```
Comment (1hitsong) [inline README.md]: >  open an issue

Let's link this direct to feature request: https://github.com/jellyfin/jellyfin-roku/issues/new?assignees=&labels=feature&template=feature_request.md&title=

Comment (cewert) [reply README.md]: Link added 👍

**Review by 1hitsong — APPROVED** [review]

**Pull request merged**